react/lib/datepicker/README.md

```diff
@@ -0,0 +1,32 @@
+## DatePicker
```

Comment:
Rather than updating the app's readme, I made a new readme for the library component. It seems more appropriate for the standalone reusable component to have its own set of instructions on how to use it and develop for it